src/test/docker/centos7-base/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM centos:7
+
+LABEL maintainer="Tom Scogland <scogland1@llnl.gov>"
+
+# add EPEL so we don't have to build everything by hand
+RUN yum -y update \
+ && yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+ && yum -y update \
+ && yum -y install \
+      which \
+      sudo \
+      git \
+      wget \
+      vim-minimal \
+      autoconf \
+      automake \
+      libtool \
+      gcc \
+      gcc-c++ \
+      file \
+      make \
+      munge \
+      munge-devel \
+      coreutils \
+      ccache \
+      cppcheck \
+      czmq-devel \
+      hwloc-devel \
+      jansson-devel \
+      sqlite-devel \
+      uuid-devel \
+      libuuid-devel \
+      libfaketime \
+      libsodium-devel \
+      lua \
+      lua-devel \
+      lua-posix \
+      mpich-devel \
+      pkgconfig \
+      python-devel \
+      python-cffi \
+      ruby \
+      sqlite \
+      yaml-cpp-devel \
+      valgrind \
+      man-db \
+ && yum clean all
+
+# The cmake from yum is incredibly ancient, download a less ancient one
+RUN wget -q --no-check-certificate https://cmake.org/files/v3.10/cmake-3.10.1-Linux-x86_64.tar.gz\
```

Comment:
Why no checking of the certificate?  I presume this is more for compatibility when building at LLNL than when building on Travis.

Reply:
I'm not sure, but your supposition sounds correct.